src/Symfony/Component/Intl/Data/Bundle/Compiler/GenrbCompiler.php

```diff
@@ -34,13 +34,13 @@ class GenrbCompiler implements BundleCompilerInterface
      */
     public function __construct(string $genrb = 'genrb', string $envVars = '')
     {
-        exec('which '.$genrb, $output, $status);
+        $this->genrb = ($envVars ? $envVars.' ' : '').$genrb;
```

Comment:
I understand that you want to also escape env vars, but the way this method is constructed makes it impossible.
We'd need to turn $envVars into an array to escape each value individually.